docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -293,26 +361,60 @@ main(){
         esac
     done
 
-    if [[ ${MINIKUBE_PROFILE} == "miniziti" ]]; then
-        banner
-    else
-        # sanity check the profile name input
-        if [[ ${MINIKUBE_PROFILE} =~ ^- ]]; then
-            # in case the arg value is another option instead of the profile name
-            logError "--profile needs a profile name not starting with a hyphen"
-            _usage; exit 1
-        fi
-        # print the alternative profile name if not default "miniziti"
-        banner "$MINIKUBE_PROFILE"
+    : "${ZITI_NAMESPACE:=${MINIKUBE_PROFILE}}"
+
+    MINIZITI_PROFILE_MARKER="miniziti-controller.$MINIKUBE_PROFILE."
+    MINIZITI_INGRESS_ZONE="$MINIKUBE_PROFILE.ziti"
+    state_dir="$(makeMinizitiXDGStateDir "$DETECTED_OS")"
+    profile_dir="$state_dir/profiles/${MINIKUBE_PROFILE}"
+    identities_dir="$profile_dir/identities"
+
+    if (( DO_ZITI_LOGIN )); then
+        miniziti_login
+        exit 0
     fi
+    if (( SHOW_ADMIN_CREDS )); then
+        showAdminCreds
+        exit 0
+    fi
+
+    if (( RUN_ZITI_CLI )); then
+        ziti_wrapper  "${ziti_cli_args[@]}"
+        exit 0
+    fi
+
+
+    banner "$MINIKUBE_PROFILE"
 
-    # delete and exit if --delete
     (( DELETE_MINIZITI )) && {
         deleteMiniziti 10
+
+        if (( MINIZITI_MODIFY_HOSTS )) && grep -q "$MINIZITI_PROFILE_MARKER" "$HOSTS_FILE"; then
+            checkSudoRequired
+            cleanHosts
+        fi
+
+        if [[ -d "$profile_dir" ]]; then
```

Comment:
It will help readability if we stick to some capitalization convention for var and function names. It doesn't have to be uppercase, just consistent. I'm open to separate conventions for heritable, overrideable, global, local, iterable, etc., but that feels like overkill.

Reply:
I'll capitalize variables in `main()` that are intended as `readonly` pseudo constants.

Reply:
Should be resolved.